docs/Documentation/Drivers/Image/OutputFormatting.md

```diff
@@ -20,42 +18,91 @@ VIC allows the user to specify exactly which output files to create and which va
 ```
 # Output File Contents
 OUTFILE	_prefix_
-OUTVAR	_varname_	[_aggtype_]
-OUTVAR	_varname_	[_aggtype_]
-OUTVAR	_varname_	[_aggtype_]
+OUTFREQ         _freq_          _VALUE_
+HISTFREQ        _freq_          _VALUE_
```

Comment:
Ah they are described below (although still need to be included in global parameter documentation)